src/azure-cli/azure/cli/command_modules/monitor/_help.py

```diff
@@ -726,7 +745,12 @@
         az monitor autoscale update -g {myrg} -n {autoscale-name} \\
           --remove-action email bob@contoso.com
   - name: Update autoscale settings. (autogenerated)
-    text: az monitor autoscale update --count 3 --email-administrator true --enabled true --max-count 5 --min-count 2 --name MyAutoscaleSettings --resource-group MyResourceGroup --tags key[=value]
+    text: |
+        az monitor autoscale update --count 3 --email-administrator true --enabled true --max-count 5 --min-count 2 --name MyAutoscaleSettings --resource-group MyResourceGroup --tags key[=value]
+    crafted: true
+  - name: Update autoscale settings. (autogenerated)
+    text: |
+        az monitor autoscale update --name MyAutoscaleSettings --resource-group MyResourceGroup --set retentionPolicy.days=365 --set retentionPolicy.days=365 --set retentionPolicy.days=365 --subscription MySubscription
```

Comment:
This one. Three same `--set`. Is it able to remove this example manually first?

Reply:
Seems an aladdin generated problematic example. Of course we can

Reply:
Removed